src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Errors/CoreErrors.cs

```diff
@@ -0,0 +1,41 @@
+using System.Net;
+using Altinn.Authorization.ProblemDetails;
+
+namespace Altinn.AccessManagement.Core.Errors;
+
+/// <summary>
+/// Validation errors for the Access Management.
+/// </summary>
+public static class CoreErrors
```

Comment:
Er en eksisterende ValidationErrors som har samme domainName "AM" og error codes 1-5. Usikker på om dette da betyr at man får duplikat error id'er ut i API men for forskjellige feil 🤔

Reply:
hmmmmmmmm, godt spørsmål